ert3/engine/_record.py

```diff
@@ -5,13 +5,23 @@
 import ert3
 
 
-def load_record(workspace: Path, record_name: str, record_file: Path) -> None:
-    with open(record_file, "r") as f:
-        raw_ensrecord = json.load(f)
+def load_record(
+    workspace: Path, record_name: str, record_file: Path, record_type: str = "normal"
```

Comment:
I think we should aim at making the `record_type` be an enum from `ert.data` to avoid the usage of magic strings.